CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+<a name="v3.0.0"></a>
+## v3.0.0 (2018-01-04)
+
+
+#### Bug Fixes
+
+*   Fixed errors treatment ((3862cc69))
+
+#### Features
+
+* **Holder**  Add Holder resource and resolves Holder params. ((9f84f793))
+* **Order**
+  *  map attributes on order creation ((8f9755ef))
+  *  add attribute paid to order creation mock ((2f01ad91))
+  *  remove shipping address attributes from populate ((7eda9b6f))
```

Comment:
I think you are also returning the `escrow` now, right? Is it something that should be here?

Reply:
No, the escrow was added recently to SDK-Java. The SDK-PHP has the escrow resource a few months ago.